html/dom/elements-metadata.js

```diff
@@ -18,6 +18,11 @@ var metadataElements = {
       defaultVal: "",
       invalidVal: ""
     },
+    scope: "string",
+    updateViaCache: {
+      type: "enum",
+      keywords: ["imports", "all", "none"]
```

Comment:
You'll want to add `defaultVal` here too.

This is used by http://w3c-test.org/html/dom/reflection-metadata.html